src/Employer/Esfa.Recruit.Azure.Resources/azuredeploy.json

```diff
@@ -396,6 +404,18 @@
             {
               "name": "Slack:WebhookUrl",
               "value": "[parameters('SlackWebhook')]"
+            },
+            {
+              "name": "Geocode:PostcodesIoUrl",
```

Comment:
I think the postcodes.io url can go in appSettings as it is not anything sensitive and would remove the work needed in maintaining these tokens and also makes the user secrets and developer setup a little lighter..

Reply:
👍